<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MortgageWindow</class>
 <widget class="QDialog" name="MortgageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>241</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="applyBtn">
      <property name="text">
       <string>Apply Mortgage</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="payBtn">
      <property name="text">
       <string>Pay Mortgage</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="mortgageList">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Comfirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="mortgageMessage">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Please select a asset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
